src/osut/osut.py

```diff
@@ -3205,7 +3205,8 @@ def uniques(pts=None, n=0) -> openstudio.Point3dVector:
     try:
         n = int(n)
     except:
-        return oslg.mismatch("n unique points", n, int, mth, CN.DBG, v)
+        oslg.mismatch("n points", n, int, mth, CN.DBG)
+        n = 0
```

Comment:
There are 3 _OSut_ functions that take in `n` as argument:

- `uniques(pts, n=0)`:
- `collinear(pts, n=0)`
- `noncollinears(pts, n=0)`

... where `n` is an (optional) user-requested number of points to return. For instance, a user may request the first 3 points of a hexagon (6 points) to determine a plane equation.

Unfortunately, all three functions did not share the same validation checks and treatment of `n` - an inconsistency and source of confusion. Furthermore, DEBUG or ERROR messages shouldn't be logged when users incorrectly guess in advance what the admissible range of points may be. So these changes:

- reduce the volume a bit
- harmonize the treatment of `n` across all three methods
- introduces stress tests